CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Show paths to config files when configuration errors occur
 
+* Added optional depth argument to `descendants`, using existing functions;
```

Comment:
Similar to my comment about the commit message in the first commit, I think this needs to say that it's about the revset function. You can find inspiration from when the release notes for when `ancestors()` gained its `depth` argument: https://github.com/martinvonz/jj/releases/tag/v0.10.0